### PR TITLE
[dsbswap.eth] Add custom domain for your space

### PR DESCRIPTION
**Pull request by DSB-Team**
[dsbswap.eth] Add custom domain for your space

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

**Comment by DSB-Team** [comment]
> There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

Yes, i just resolved.

File: spaces/domains.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "vote.dsbswap.com":"dsbswap.eth",
```
Comment (gokaiorg) [inline spaces/domains.json:2]: Please add a space here.
```suggestion
  "vote.dsbswap.com": "dsbswap.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by DSB-Team — COMMENTED** [review]
Added a space

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**